docs/src/content/hardhat-runner/docs/guides/test-contracts.md

```diff
@@ -27,18 +27,30 @@ For our first test we’ll deploy the `Lock` contract and assert that the unlock
 ```tsx
 import { expect } from "chai";
 import hre from "hardhat";
+import { buildModule } from "@nomicfoundation/hardhat-ignition/modules";
 import { time } from "@nomicfoundation/hardhat-toolbox/network-helpers";
 
+const LockModule = buildModule("LockModule", (m) => {
```

Comment:
Same as the sample projects: I don't think this is adding value, as we aren't really testing the deployment. Instead, it will create confusion and the perception that Ignition adds more complexity.

We should probably add a section of "Testing your deployment" to the deployments guide instead.

Reply:
I have reverted the contract testing guide.